dnf/cli/commands/repoquery.py

```diff
@@ -90,7 +90,7 @@ class RepoQueryCommand(commands.Command):
     """A class containing methods needed by the cli to execute the repoquery command.
     """
 
-    aliases = ('repoquery',)
+    aliases = ('repoquery', 'deplist')
```

Comment:
@j-mracek , has yum repoquery really `--deplist` option? If so then it would be easier to have it here otherwise it should be rather another command file. I don't know if architecture of yum wrapper allows adding custom commands. Does it, @mhatina ?